env-var-docs/env_var_docs/schema/README.md

```diff
@@ -0,0 +1,97 @@
+# Environment variable documentation file format
+
+We maintain a [json-schema](https://json-schema.org/) that defines the expected
+structure of env-var-docs.json in [schema.json](./schema.json).
+
+The env-var-docs.json file contains key-value pairs. Each key is either a group
+of environment variables or an environment variable referenced in
+application.conf.
+
+Groups have the following fields:
+
+- `group-description`: A human-readable description of what the group
+  configures.
+- `members`: An object containing groups or variables.
```

Comment:
**question (blocking):** thoughts on renaming `members` -> `variables`?

Reply:
Because a group can contain another group, I think members is a better name otherwise 'variable' would have two meanings

Reply:
gotcha, sounds good